### ISSUE TITLE
Human Immunoglobulin(s)

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#157]
Does it make sense to have a singular and plural version of these concepts:

CONCEPT_ID|CONCEPT_CODE|CONCEPT_NAME
--|--|--
44012674 | OMOP1131583 | Human Immunoglobulins
36878747 | OMOP992262 | human immunoglobulin

**Comment by aostropolets** [comment]
Sure it doesn't :)
Once we've gathered a sufficient pool of those, we'll fix them at once, so please keep sending all that you find :)

**Comment by ericaVoss** [comment]
Just making a note that this specific example still exists in 20180609, I know you may not have worked on it yet.

```sql
SELECT *
FROM OMOP_VOCABULARY_20180609.dbo.CONCEPT
WHERE CONCEPT_ID IN (
44012674,36878747
)
ORDER BY 1
```

**Comment by aostropolets** [comment]
@ericaVoss  Please check :)

**Comment by ericaVoss** [comment]
Hahaha, can I check the next time I download a Vocab?  I'm still on 20180609

**Comment by clairblacketer** [comment]
Just making a note that I still see this using Erica's code in vocab 20181119

**Comment by aostropolets** [comment]
Non-standard, invalid, remapped

**Comment by clairblacketer** [comment]
thanks Anna! I must have missed that as I looked through it :)

**Comment by clairblacketer** [comment]
@aostropolets yep I looked again and I was wrong -this issue was fixed and I think it can be closed
